.gitmodules

```diff
@@ -4,9 +4,6 @@
 [submodule "Quartz/ThirdParty/sol2"]
 	path = Quartz/ThirdParty/sol2
 	url = https://github.com/ThePhD/sol2.git
-[submodule "Quartz/ThirdParty/SDL2"]
-	path = Quartz/ThirdParty/SDL2
-	url = https://github.com/spurious/SDL-mirror.git
```

Comment:
Why has the SDL2 submodule been deleted?

Reply:
Git was being a right old pain, so we removed it and added it again... and turns out Git is still being a pain...

Reply:
I think that disappeared in a merge from DEV, I thought you guys did it 😂